src/BugsnagUnity/Client.cs

```diff
@@ -156,14 +167,20 @@ void Notify(string condition, string stackTrace, LogType logType)
                     Notify(new Exception[] { exception }, exception.HandledState, null, logType);
                 }
             }
-            else if (logType.IsGreaterThanOrEqualTo(Configuration.BreadcrumbLogLevel))
+            else if (ShouldLeaveLogBreadcrumb(logType))
             {
                 Breadcrumbs.Leave(logType.ToString(), BreadcrumbType.Log, new Dictionary<string, string> {
-          { "message", condition },
-        });
+                    { "message", condition },
+                });
             }
         }
 
+        private bool ShouldLeaveLogBreadcrumb(LogType logType)
```

Comment:
This should probably also belong on `Configuration`, since its the source of all of the logic used here.

Reply:
done